deny.toml

```diff
@@ -1,87 +1,47 @@
-# cargo-deny configuration for license enforcement and dependency management
+# cargo-deny configuration for rust2mojo
+
+[graph]
+all-features = false
+no-default-features = false
+
+[output]
+feature-depth = 1
+
+[advisories]
+ignore = []
+
+[licenses]
+allow = [
+    "MIT",
+    "Apache-2.0",
+]
+confidence-threshold = 0.8
+exceptions = [
+    { allow = ["Unicode-3.0"], crate = "unicode-ident" },
+]
+
+[licenses.private]
+ignore = false
+registries = []
 
 [bans]
-# Lint level for when multiple versions of the same crate are detected
 multiple-versions = "warn"
-# Lint level for when a crate version requirement is `*`
 wildcards = "allow"
-# The graph highlights used when creating dotgraphs for crates
 highlight = "all"
-# The default lint level for `default` fields in the `[bans.deny]` section
 workspace-default-features = "allow"
-# The default lint level for `external-default-features` fields in the `[bans.deny]` section
 external-default-features = "allow"
-# List of crates that are allowed. Use with care!
 allow = []
-# List of crates to deny
 deny = []
-# Certain crates/versions that will be skipped when doing duplicate detection.
 skip = []
-# Similarly to `skip` allows you to skip certain crates from being checked. Unlike `skip`,
-# `skip-tree` skips the crate and all of its dependencies entirely.
 skip-tree = []
 
-[licenses]
-# The confidence threshold for detecting a license from a license text.
-# The higher the value, the more closely the license text must match the
-# canonical license text of a valid SPDX license file.
-confidence-threshold = 0.8
-# List of explicitly allowed licenses - STRICT: Only Apache-2.0 and MIT
-allow = [
```

Comment:
If we don't include the allowlist, we won't be able to block noncompliant changes.